pixeltable/store.py

```diff
@@ -186,9 +193,9 @@ def add_column(self, col: catalog.Column, conn: sql.engine.Connection) -> None:
         message).
         """
         assert col.is_stored
-        stmt = sql.text(f'ALTER TABLE {self._storage_name()} ADD COLUMN {col.storage_name()} {col.col_type.to_sql()}')
-        log_stmt(_logger, stmt)
-        conn.execute(stmt)
+        clause = sql.text(f'ALTER TABLE {self._storage_name()} ADD COLUMN {col.storage_name()} {col.col_type.to_sql()}')
```

Comment:
Why the rename? This is a statement, not a clause, from a SQL perspective.

Reply:
There's another variable named `stmt` later in the same function, which takes an unrelated type (a `tuple`). mypy doesn't like it (rightly so, IMO) when the same variable is reused in the same scope for unrelated types.

Reply:
Oh right. Probably best to call those stmt_str (they're strings).

Reply:
I found a different, much better way to address this. See the updated PR (posting in a few minutes)